boards/openmote-b/include/periph_conf.h

```diff
@@ -148,6 +148,24 @@ static const spi_conf_t spi_config[] = {
 #define RADIO_IRQ_PRIO      1
 /** @} */
 
+/**
+ * @name RTT configuration
+ * @{
+ */
+#define RTT_DEV             SMWDTHROSC
+#define RTT_IRQ             SM_TIMER_ALT_IRQn
+#define RTT_IRQ_PRIO        1
+#define RTT_ISR             isr_sleepmode
+#define RTT_MAX_VALUE       (0xffffffff)
+#if SYS_CTRL_OSC32K_USE_XTAL
+/* Frequency of XOSC off by default */
+#define RTT_FREQUENCY       (32768U)    /* in Hz. For changes see `rtt.c` */
+#else
+/* Frequency of RCOSC on by default */
+#define RTT_FREQUENCY       (32753U)    /* in Hz. For changes see `rtt.c` */
+#endif
+/** @} */
+
```

Comment:
Does it make sense to make this board-specific?
It could as well go to `cpu/cc2538/include/periph_cpu.h`.

Reply:
Makes sense!